### PR TITLE
adds drop command as a valid ddl command

### PR DESCRIPTION
**Pull request by benwulfe**
fixes a related bug in spannercommandtextbuilder
adds tests

### CODE REVIEW

File: apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/SpannerCommandTextBuilder.cs [hunk 1]
```diff
@@ -34,7 +35,9 @@ public sealed class SpannerCommandTextBuilder
         private const string AlterCommand = "ALTER";
         private const string CreateCommand = "CREATE";
         private const string DropCommand = "DROP";
+        //TODO(benwu): verify if whitespace between CREATE/DROP and DB is allowed.
         private const string CreateDatabaseCommand = "CREATE DATABASE";
+        private const string DropDatabaseCommand = "DROP DATABASE";
```
Comment (jskeet) [inline apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/SpannerCommandTextBuilder.cs]: Should you have a space after this? At the moment it *looks* like a DDL of `DROP DATABASEFOO` will have the same effect as `DROP DATABASE FOO` and presumably it shouldn't.

(Ditto CREATE)

Comment (benwulfe) [inline apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/SpannerCommandTextBuilder.cs]: yep --- and I should have a bunch of negative tests as well.

Comment (benwulfe) [inline apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/SpannerCommandTextBuilder.cs]: also its probably safer to fail on these sorts of statements:

DROP DATABASE NAMEWITH ANACCIDENTALSPACE

than to drop "NAMEWITH"

**Comment by benwulfe** [comment]
added space and more test coverage.

**Pull request merged**